addons/website/views/snippets/s_ecomm_banner_full.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_ecomm_banner_full" name="Ecomm Banner Full">
+    <section
+        class="s_ecomm_banner_full o_cc o_cc5 oe_img_bg pt128 pb24"
+        style="background-image: url('/web/image/website.s_ecomm_banner_full_default_image'); background-position: 50% 75%;"
+        >
+        <div class="o_we_bg_filter bg-black-25"/>
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8">
```

Comment:
I think that the data-row-count should be 11

Reply:
Good catch ! I've tested with 11 but it was too high, so I've defined it to 10 which is perfect 👍